### PR TITLE
fix(local-process) output Ghost error message when process exits

### PR DESCRIPTION
**Pull request by vikaspotluri123**
Tests are broken, but I'd like to get a logic review first 😄

### CODE REVIEW

**Comment by vikaspotluri123** [comment]
One concern I have with this implementation is I'm not sure how the pipe will work after disconnecting the CP - will it still be connected on the CLI side (I would presume no), and where will stderr go on the cp-side?

File: lib/utils/local-process.js [hunk 1]
```diff
@@ -65,6 +74,9 @@ Ensure the content folder has correct permissions and try again.`));
 
                 /* istanbul ignore else */
                 if (msg.started) {
+                    buffer = '';
```
Comment (acburdine) [inline lib/utils/local-process.js:77]: do we need to reset buffer here? seems like it shouldn't matter 🤔

**Comment by acburdine** [comment]
> One concern I have with this implementation is I'm not sure how the pipe will work after disconnecting the CP - will it still be connected on the CLI side (I would presume no), and where will stderr go on the cp-side?

it will likely still be connected, we might need to call `cp.stderr.destroy()` to close the stream. On the cp side, I _believe_ the stderr will still be ignored? According to the node [stream docs](https://nodejs.org/api/stream.html#stream_readable_destroy_error):
> After this call, the readable stream will release any internal resources and subsequent calls to push() will be ignored.

**Comment by acburdine** [comment]
@vikaspotluri123 created a small repro case to answer your above questions: https://gist.github.com/acburdine/f6cf5241281ee81bfee1bab4214363de

**Comment by vikaspotluri123** [comment]
Forgot to add the issue ref - refs #1201

**Pull request merged**